.gitignore

```diff
@@ -1,5 +1,6 @@
 /es
 /node_modules
 /lib
+/typings
```

Comment:
I think that change is no longer necessary - we could use the _typings_ folder to store definitions for external modules (e. g. I think it's a good place to store that file: <https://github.com/trojanowski/react-apollo-hooks/pull/46/files#diff-352f02e2ca9a9158d200b6b51f0e1c4c>). The same for _.prettierignore_
```suggestion
```